Change #include from <...> to "..." for dwarf.h and libdwarf.h to use
the version from the libdwarf source tree.  Otherwise, this can break
the build if there is a different version of these files in
/usr/include.


diff -Naurb dwarf-20150915.orig/libdwarf/dwarf_addr_finder.c dwarf-20150915/libdwarf/dwarf_addr_finder.c
--- dwarf-20150915.orig/libdwarf/dwarf_addr_finder.c	2015-09-03 15:29:57.000000000 -0500
+++ dwarf-20150915/libdwarf/dwarf_addr_finder.c	2016-11-12 21:11:59.364846438 -0600
@@ -36,8 +36,8 @@
 #ifdef HAVE_ELF_H
 #include <elf.h>
 #endif
-#include <dwarf.h>
-#include <libdwarf.h>
+#include "dwarf.h"
+#include "libdwarf.h"
 #include "dwarf_base_types.h"
 #include "dwarf_alloc.h"
 #include "dwarf_opaque.h"
diff -Naurb dwarf-20150915.orig/libdwarf/dwarf_incl.h dwarf-20150915/libdwarf/dwarf_incl.h
--- dwarf-20150915.orig/libdwarf/dwarf_incl.h	2015-09-15 11:36:08.000000000 -0500
+++ dwarf-20150915/libdwarf/dwarf_incl.h	2016-11-12 21:11:25.033068009 -0600
@@ -49,8 +49,8 @@
 #endif
 
 #include <limits.h>
-#include <dwarf.h>
-#include <libdwarf.h>
+#include "dwarf.h"
+#include "libdwarf.h"
 
 #include "dwarf_base_types.h"
 #include "dwarf_alloc.h"
